assets/src/stories-editor/components/higher-order/with-right-click-handler.js

```diff
@@ -55,20 +69,34 @@ const applyWithSelect = withSelect( ( select, props ) => {
 		if ( toolBar ) {
 			toolBarHeight = toolBar.clientHeight;
 		}
-		const relativePositionX = event.clientX - wrapperDimensions.left;
-		const relativePositionY = event.clientY - wrapperDimensions.top - toolBarHeight;
+
+		let eventX = 0;
+		let eventY = 0;
+
+		if ( isContextmenu ) {
+			eventX = event.clientX;
+			eventY = event.clientY;
+		} else if ( isKeydown ) {
```

Comment:
Why is this a else if? Is there a chance that at this point in the code, it would not be a keydown event?

Reply:
Not currently. But I like to be on the safe side and it reads more cleanly – you know when reading the code what situation is handled by this branch – could also be solved by a comment of course. Should I change it to a `switch` perhaps?

Reply:
Fixed in 3fa7dd1.